iceoryx_posh/source/roudi/roudi_config_toml_file_provider.cpp

```diff
@@ -28,12 +28,12 @@ namespace iox
 {
 namespace config
 {
-TomlRouDiConfigFileProvider::TomlRouDiConfigFileProvider(CmdLineParserConfigFileOption& cmdLineParser)
+TomlRouDiConfigFileProvider::TomlRouDiConfigFileProvider(config::CmdLineArgs_t& cmdLineArgs)
 {
     /// don't print additional output if not running
-    if (cmdLineParser.getRun())
+    if (cmdLineArgs.run)
     {
-        if (cmdLineParser.getConfigFilePath().size() == 0)
+        if (cmdLineArgs.configFilePath.size() == 0)
```

Comment:
In general it is better to write here `cmdLineArgs.configFilePath.empty()`.

Reply:
Fixed